presto-main/src/test/java/com/facebook/presto/block/AbstractTestBlock.java

```diff
@@ -86,6 +90,31 @@
         }
     }
 
+    protected List<Block> splitBlock(Block block, int count)
```

Comment:
There are existing code that splits block into 4. I changed those code to call this method.